### PR TITLE
feature(flow): Add return typings

### PR DESCRIPTION
**Pull request by captainsafia**
I think I configured this right?

### CODE REVIEW

**Comment by jdetle** [comment]
Thanks @captainsafia!!!

File: src/terminals.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 // @flow
 
 import { ajax } from 'rxjs/observable/dom/ajax';
+import Observable from 'rxjs/Observable';
```
Comment (rgbkrk) [inline src/terminals.js]: this looks a tiny bit suspect, like it needs to be `rxjs/Observable`. This primarily has impact on case sensitive file systems (non-mac). Separately, you may be able to `import type`.

Comment (rgbkrk) [inline src/terminals.js]: Ha, I just used the upper casing here too.

Comment (captainsafia) [inline src/terminals.js]: I think the uppercase should work fine. It's written as such in [the RxJS docs](http://reactivex.io/rxjs/file/es6/Observable.js.html).

Comment (rgbkrk) [inline src/terminals.js]: Ok cool.

**Pull request merged**